templates/repo/issue/view_content/comments.tmpl

```diff
@@ -6,7 +6,7 @@
 	 5 = COMMENT_REF, 6 = PULL_REF, 7 = COMMENT_LABEL, 12 = START_TRACKING,
 	 13 = STOP_TRACKING, 14 = ADD_TIME_MANUAL, 16 = ADDED_DEADLINE, 17 = MODIFIED_DEADLINE,
 	 18 = REMOVED_DEADLINE, 19 = ADD_DEPENDENCY, 20 = REMOVE_DEPENDENCY, 21 = CODE,
-	 22 = REVIEW, 23 = ISSUE_LOCKED, 24 = ISSUE_UNLOCKED -->
+	 22 = REVIEW, 23 = ISSUE_LOCKED, 24 = ISSUE_UNLOCKED 25 = PROJECT BOARD CHANGED -->
```

Comment:
https://github.com/go-gitea/gitea/pull/8346/files#diff-9dbad6d760dba020a994d9ecc3fd09dcR86

-> Project changed is not handled!!